src/components/Toggle/docs/Toggle.stories.mdx

```diff
@@ -0,0 +1,43 @@
+import { ItemWrapper } from '../../../../docs/components/ItemWrapper';
+import { StyledSystemLinks } from '../../../docs/StyledSystemLinks';
+import { Toggle } from '../Toggle';
+import { TogglePropsTable } from './TogglePropsTable';
+import { Canvas, Story, ArgsTable, Meta } from '@storybook/addon-docs';
+
+<Meta title="Components/Toggle" component={Toggle} />
+
+export const Template = args => <Toggle {...args} />;
+
+# Toggle
+
+Toggle is a control that is used to quickly switch between two possible states. Toggles are only used for these binary
+actions that occur immediately after the user “flips” the toggle switch. They are commonly used for “on/off” switches.
+
+<ItemWrapper gridTemplate="auto auto auto auto 1fr">
+    <Toggle defaultChecked />
+    <Toggle />
+    <Toggle error defaultChecked />
+    <Toggle disabled />
+</ItemWrapper>
+
+## Properties
+
+<ArgsTable of={Toggle} />
+<StyledSystemLinks component="Toggle" supportedProps={['margin']} />
+
+## Behaviour
+
+-   Use toggle to swap the state of a functionality or preference (usually between „on“ and „off“)
+-   For multi-state scenarios consider radio buttons or checkboxes
+-   Toggles should take affect automatically (without saving)
+
+## Technical Information
+
+Inside the toggle component is a checkbox and therefore the api and usage works similarly. For example, instead
+of passing the `value` prop, the `checked` prop must be used instead.
+
+## Playground
```

Comment:
Can we name this **Stories** as `<Canvas>` and `<Story>` components don't exactly provide a Playground for us, as before, that's replaced with Canvas tab now :)